actors/builtin/multisig/multisig_actor.go

```diff
@@ -368,11 +363,30 @@ func (a Actor) approveTransaction(rt vmr.Runtime, txnID TxnID, proposalHash []by
 }
 
 func (a Actor) validateSigner(rt vmr.Runtime, st *State, address addr.Address) {
-	if !st.isSigner(address) {
+	if !a.isSigner(rt, st, address) {
 		rt.Abortf(exitcode.ErrForbidden, "party not a signer")
 	}
 }
 
+func (a Actor) isSigner(rt vmr.Runtime, st *State, address addr.Address) bool {
```

Comment:
Nit: neither this nor `validateSigner` need to be methods of `Actor` since they don't reference it at all. This is the general case for all actors: methods are either exported or not methods.

Prefer to make these free functions, and also move them to the bottom of this file so all the exported methods come first.